src/mappings/poolXYKHandler.ts

```diff
@@ -49,7 +49,7 @@ export async function handleXYKPools(block: SubstrateBlock): Promise<void> {
     //todo rework to liquidity proxy quote
     let reserves;
     try {
-        reserves = await api.query.poolXyk.reserves.entries(XOR)
+        reserves = await api.query.poolXYK.reserves.entries(XOR)
```

Comment:
Where is `api` defined? I cannot see the import of `api`  in this file

Reply:
sora-subquery/node_modules/@subql/types/dist/global.d.ts